bundle/jsky.app.ot/src/main/scala/edu/gemini/catalog/ui/package.scala

```diff
@@ -228,31 +228,38 @@ case class IdColumn(title: String) extends CatalogNavigatorColumn[String] {
 object GuidingQualityColumn {
   implicit val analysisOrder:Order[AgsAnalysis] = Order.orderBy(_.quality)
 
+  def validatableGuideProbe(gp: GuideProbe): Option[ValidatableGuideProbe] =
+    gp match {
+      case vgp: ValidatableGuideProbe => Some(vgp)
+      case _                          => None
+    }
+
   // Calculate the guiding quality of the target considering only its magnitude
   // while ignoring reachability.
-  def target2Analysis(info: Option[ObservationInfo], t: SiderealTarget): Option[AgsAnalysis] =
+  def target2Analysis(info: Option[ObservationInfo], gpo: Option[GuideProbe], t: SiderealTarget): Option[AgsAnalysis] =
     for {
       o   <- info
+      vgp <- gpo.flatMap(validatableGuideProbe) orElse o.guideProbe
```

Comment:
A key difference from before is that we use the appropriate guide probe for the magnitude analysis.